docs/development.md

```diff
@@ -25,7 +25,17 @@ Simply add this require statement to your spec_helper:
 require 'solidus_starter_frontend/factories'
 ```
 
+### Running the extension in development
+
+In order to run the extension, you can either 1) run the sandbox script to
+generate the sandbox app or 2) add the extension as a gem in a Rails app.
+
 ### Running the sandbox
```

Comment:
Minor suggestion. Should this title be `####` (4 `#`) as it's within `Running the extension in development` (which is 3 `#`)?

Reply:
Yeah, I intentionally kept the two headings at a higher level for aesthetic purposes. I can move them lower.

Reply:
Maybe you're right, I didn't check the design :slightly_smiling_face:

Reply:
Nah, I think your suggestion works better. I lowered the headings.